### DISCUSSION TITLE
Disable post nav at certain pages

### DISCUSSION THREAD

**Discussion by iamgiannisx** [razonyang/hugo-theme-bootstrap#584] [Ideas]
Is it possible to disable card-footer without change nav.html template ?

**Comment by razonyang** [comment]
No, I'll create a parameter for this.

**Comment by razonyang** [comment]
Implemented by #586. You'll need to upgrade the theme to the latest commit via `hugo mod get github.com/razonyang/hugo-theme-bootstrap@master`.

You can either set`post.nav = false` in params file or set `nav = false` in front matter. The former method disable the post navigation globally.

**Reply by iamgiannisx** [reply]
Thanks! working as expected
